### PR TITLE
J85: + enum-type-to-podcasts-table

### PR DESCRIPTION
**Pull request by Sayonarko**

### CODE REVIEW

File: backend/src/data/migrations/20210805155531_add_type_to_podcasts.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Knex } from 'knex';
+
+const PodcastsType = {
```
Comment (ghost) [inline backend/src/data/migrations/20210805155531_add_type_to_podcasts.ts]:
```suggestion
const PodcastType = {
```

**Review by ghost — CHANGES_REQUESTED** [review]
don't forget to change the model, type and etc

**Review by Denidzio — DISMISSED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
Don't forget about model, type and etc

**Review by yevhenprybytok — DISMISSED** [review]

File: backend/src/validation-schemas/podcast/create-podcast/create-podcast.validation-schema.ts [hunk 2]
```diff
@@ -16,6 +17,7 @@ const podcastCreate = podcast.keys({
     }),
     Joi.any().equal(null),
   ],
+  [PodcastPayloadKey.TYPE]: Joi.string().valid(...Object.values(PodcastType)),
```
Comment (ghost) [inline backend/src/validation-schemas/podcast/create-podcast/create-podcast.validation-schema.ts]:
```suggestion
  const podcastTypes = Object.values(PodcastType);
  [PodcastPayloadKey.TYPE]: Joi.string().valid(...podcastTypes),
```

**Review by ghost — CHANGES_REQUESTED** [review]

**Pull request merged**